packages/ui/src/app/pages/Proposals/CurrentProposals.stories.tsx

```diff
@@ -102,9 +269,1182 @@ export default {
+  parameters: {
+    stakingAccountIdMemberStatus: {
+      memberId: 0,
+      confirmed: { isTrue: false },
+      size: 0,
+    },
+    createProposalFailure: 'It failed 🙀',
+  },
+
+  play: async ({ args, canvasElement, step }) => {
+    const modal = withinModal(canvasElement)
+    await completeForms(canvasElement, step)
+    await userEvent.click(modal.getByText('Sign transaction and Bind Staking Account'))
+    await userEvent.click(await modal.findByText('Sign transaction and Create'))
+
+    expect(await modal.findByText('It failed 🙀'))
+    within(document.body).getByText('Transaction failed')
+
+    expect(args.onAddStakingAccountCandidate).toHaveBeenCalled()
+    expect(args.onConfirmStakingAccount).toHaveBeenCalled()
+    expect(args.onCreateProposal).toHaveBeenCalled()
+    expect(args.onChangeThreadMode).not.toHaveBeenCalled()
+  },
+}
+
+export const ConfirmAccountThenCreateProposalFailure: Story = {
+  parameters: {
+    stakingAccountIdMemberStatus: {
+      memberId: 0,
+      confirmed: { isTrue: false },
+      size: 1,
+    },
+    createProposalFailure: 'It failed 🙀',
+  },
+
+  play: async ({ args, canvasElement, step }) => {
+    const modal = withinModal(canvasElement)
+    await completeForms(canvasElement, step)
+    await userEvent.click(await modal.findByText('Sign transaction and Create'))
+
+    expect(await modal.findByText('It failed 🙀'))
+    within(document.body).getByText('Transaction failed')
+
+    expect(args.onAddStakingAccountCandidate).not.toHaveBeenCalled()
+    expect(args.onConfirmStakingAccount).toHaveBeenCalled()
+    expect(args.onCreateProposal).toHaveBeenCalled()
+    expect(args.onChangeThreadMode).not.toHaveBeenCalled()
+  },
+}
+
+export const CreateProposalFailure: Story = {
+  parameters: {
+    createProposalFailure: 'It failed 🙀',
+  },
+
+  play: async ({ args, canvasElement, step }) => {
+    const modal = withinModal(canvasElement)
+    await completeForms(canvasElement, step)
+    await userEvent.click(await modal.findByText('Sign transaction and Create'))
+
+    expect(await modal.findByText('It failed 🙀'))
+    within(document.body).getByText('Transaction failed')
+
+    expect(args.onAddStakingAccountCandidate).not.toHaveBeenCalled()
+    expect(args.onConfirmStakingAccount).not.toHaveBeenCalled()
+    expect(args.onCreateProposal).toHaveBeenCalled()
+    expect(args.onChangeThreadMode).not.toHaveBeenCalled()
+  },
+}
+
+export const ChangeThreadModeFailure: Story = {
+  parameters: {
+    changeThreadModeFailure: 'It failed 🙀',
+  },
+
+  play: async ({ args, canvasElement, step }) => {
+    const modal = withinModal(canvasElement)
+    await completeForms(canvasElement, step)
+    await userEvent.click(await modal.findByText('Sign transaction and Create'))
+    await userEvent.click(await modal.findByText('Sign transaction and change mode'))
+
+    expect(await modal.findByText('It failed 🙀'))
+    // within(document.body).getByText('Transaction failed')
```

Comment:
The `TransactionStatus` popup doesn't appear during the `tx.proposalsDiscussion.changeThreadMode`